src/routes/+layout.svelte

```diff
@@ -1,13 +1,15 @@
 <script lang="ts">
+	import type { Profile } from '$lib/types/profile';
 	import { invalidate } from '$app/navigation';
 	import { onMount } from 'svelte';
 
 	import { supabase } from '$lib/auth';
 	import Header from '$lib/components/Header.svelte';
+	import Modal from '$lib/components/Modal.svelte';
 
 	export let data;
 
-	$: user = data.user;
+	$: user = data.user as Profile;
```

Comment:
I thought `data` was properly typed by the supabase generated typings

Reply:
Not totally, it doesn't include relation (Experience, Formation etc.. )

Reply:
ok